test/Mono.Linker.Tests.Cases/Warnings/WarningSuppression/AddSuppressionsBeforeAttributeRemoval.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <linker xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../src/ILLink.Shared/ILLink.LinkAttributes.xsd">
-	<assembly fullname="test, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
+	<assembly fullname="*">
```

Comment:
Feels weird that we have to use star. There is plenty of other XML files for descriptors and XML attributes that use this same method.